nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core-api/src/main/java/org/apache/nifi/controller/flowanalysis/AnalyzeFlowRequest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.nifi.controller.flowanalysis;
+
+import org.apache.nifi.flowanalysis.AnalyzeFlowState;
+import org.apache.nifi.flowanalysis.AnalyzeFlowStatus;
+
+public class AnalyzeFlowRequest implements AnalyzeFlowStatus {
+    private final String processGroupId;
+
+    private final long submissionTime = System.currentTimeMillis();
```

Comment:
Recommend aligning the property name with the accessor name:
```suggestion
    private final long requestSubmissionTime = System.currentTimeMillis();
```

Reply:
This class is not needed any more and is going to be removed.